fiware-region-sanity-tests/commons/constants.py

```diff
@@ -88,6 +93,3 @@
 
 # ENDPOINTS
 ENDPOINT_TYPE_PUBLIC_URL = "publicURL"
-
-# TYPES
-TEST_TEXT_CONTENT_TYPE = "text/plain"
```

Comment:
Why do you delete this line, we do not need the test associated to the upload of a text file in the Object Storage?